orchestra/cmds/main.py

```diff
@@ -38,15 +39,16 @@ def __init__(self, handler=None, *args, **kwargs):
     parser_class=CustomArgumentParser,
 )
 components.install_subcommand(main_subparsers)
-dumpconfig.install_subcommand(main_subparsers)
 environment.install_subcommand(main_subparsers)
 clone.install_subcommand(main_subparsers)
-update.install_subcommand(main_subparsers)
 configure.install_subcommand(main_subparsers)
 install.install_subcommand(main_subparsers)
-clean.install_subcommand(main_subparsers)
 uninstall.install_subcommand(main_subparsers)
+clean.install_subcommand(main_subparsers)
+update.install_subcommand(main_subparsers)
+upgrade.install_subcommand(main_subparsers)
 graph.install_subcommand(main_subparsers)
 shell.install_subcommand(main_subparsers)
 ls.install_subcommand(main_subparsers)
 fix_binary_archives_symlinks.install_subcommand(main_subparsers)
+dumpconfig.install_subcommand(main_subparsers)
```

Comment:
Shall we have a list and loop over it?

Reply:
I thought about it but saw not real difference in readability or maintainability, the code would get longer by a couple of lines but use fewer characters

```
cmds = [
    components,
    dumpconfig,
    ...
]

for cmd in cmds:
    cmd.install_subcommand(main_subparsers)
```

If you prefer this version I'll change it, I have no strong opinion about it.

Reply:
list ftw

Reply:
Done